app/routes/applicationForm/$applicantId/index.tsx

```diff
@@ -112,14 +119,16 @@ const profileFetcherOptions: SubmitOptions = {
 
 export const loader: LoaderFunction = async ({ request }) => {
   const universities = await getUniversities();
+  const profile = await requireProfile(request);
```

Comment:
but here we are getting the info from our database, where we save the data that we get from LinkedIn?

Reply:
The information we get from LinkedIn is being stored in the profile table, not much information is obtained through the login with LinkedIn so I decided to store it in profile, here is an example of the data that is obtained through the LinkedIn login:

<img width="913" alt="Captura de pantalla 2023-12-05 a la(s) 10 16 17 a m" src="https://github.com/wizeline/remix-project-lab/assets/58340818/7c98741e-073e-4a98-9e37-7b57ff90c7db">